### PR TITLE
Remove extra stuff and workspaces

### PR DESCRIPTION
**Pull request by theRealPadster**
WIP

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,3 +1,3 @@
 * @spicetify/marketplace
```
Comment (theRealPadster) [inline .github/CODEOWNERS:1]: I'm not sure what this line does tbh. Does it need to be adjusted as well or no?

File: package.json [hunk 2]
```diff
@@ -1,19 +1,50 @@
...
+		"lint:ci": "eslint src",
+		"type-check": "tsc --noEmit",
+		"watch": "spicetify-creator --watch",
+		"prepare": "husky install"
+	},
+	"engines": {
+		"node": ">=16"
 	},
-	"private": true,
-	"packageManager": "yarn@3.2.1",
```
Comment (theRealPadster) [inline package.json:15]: Oh, do we need this still?

File: package.json [hunk 3]
```diff
@@ -1,19 +1,50 @@
...
 	"devDependencies": {
-		"husky": "^8.0.2"
```
Comment (theRealPadster) [inline package.json:17]: And this? I just moved the package.json from marketplace to the main one and overwrote it.

File: .github/CODEOWNERS (hunk 1 above)
Comment (kyrie25) [reply .github/CODEOWNERS:1]: Nah, that just means every file in the repository is owned by the `@spicetify/marketplace` team

File: package.json (hunk 2 above)
Comment (kyrie25) [reply package.json:15]: Those can stay I think, using a new version of Yarn is always nice

File: package.json (hunk 3 above)
Comment (kyrie25) [reply package.json:17]: Husky is for linting before committing, that should stay imo.

**Comment by theRealPadster** [comment]
Does anything else need tweaking?

**Review by CharlieS1103 — APPROVED** [review]

**Pull request merged**